aclk/agent_cloud_link.c

```diff
@@ -131,6 +133,57 @@ int cloud_to_agent_parse(JSON_ENTRY *e)
 }
 
 
+static RSA *aclk_private_key = NULL;
+static int create_private_key()
+{
+    char filename[FILENAME_MAX + 1];    struct stat statbuf;
+    snprintfz(filename, FILENAME_MAX, "%s/claim.d/private.pem", netdata_configured_user_config_dir);
+
+    if (lstat(filename, &statbuf) != 0) {
+        error("Claimed agent cannot establish ACLK - private key not found '%s' failed reason=\"%s\".", filename, strerror(errno));
```

Comment:
The `error()` netdata function already prints `strerror(errno)` so it's going to print this twice.

Reply:
Fixed.